scripts/start-docker.sh

```diff
@@ -9,20 +9,34 @@
 # Get the numeric user ID of the current directory.
 uid=$(ls -nd . | awk '{ print $3 }')
 
-# Create an `olympia` user with that ID, and the current directory
-# as its home directory.
-if [[ $uid -eq 0 ]]; then
-    # Don't try and create a user with uid 0 since it will fail.
-    # Works around issue with docker for mac running containers
-    # as root and not the user. Instead we just create the olympia
-    # user since files will still be owned by the host's user
-    # due to the way osxfs works.
-    useradd -Md $(pwd) olympia
-else
-    useradd -Md $(pwd) -u $uid olympia
+# # Create an `olympia` user with that ID, and the current directory
+# # as its home directory.
+# if [[ $uid -eq 0 ]]; then
+#     # Don't try and create a user with uid 0 since it will fail.
+#     # Works around issue with docker for mac running containers
+#     # as root and not the user. Instead we just create the olympia
+#     # user since files will still be owned by the host's user
+#     # due to the way osxfs works.
+#     useradd -Md $(pwd) olympia
+# else
+#     useradd -Md $(pwd) -u $uid olympia
+# fi
+
+deps_uid=$(ls -nd . | awk '{ print $3 }')
+
+# Fix /deps/ folder so that we're able to update the image
+# with the `olympia` user
+if [[ $deps_uid -ne $uid ]]; then
```

Comment:
I guess this doesn't hurt but why would the user be wrong?

Reply:
With the recent changes, this entrypoint will hopefully be removed - I haven't done that yet while the testing isn't done.

So, don't worry about that just yet.

Reply:
I put the `chown` in the Dockerfile instead which enforces the correct user assignment.